R/projection.R

```diff
@@ -55,8 +47,40 @@ calc_svd_ops <- function(V_row, dims = NULL) {
 #'
 #' @param scaling dso$st$scaling object containing sinkhorn scaling result
 #' @param ops svd result for the matirx
+#' @param dim how many dimension we what
 #' @return proj object
-svd_project_with_ops <- function(scaling, ops) {
+svd_project_with_ops <- function(scaling, ops, dims = NULL) {
```

Comment:
Now scaling object is not the scaling object. How do you think is is a correct naming?